src/nim_status_client.nim

```diff
@@ -27,6 +27,23 @@ when defined(macosx) and defined(arm64):
 when defined(windows):
     {.link: "../status.o".}
 
+when defined(USE_QML_SERVER):
+  # get the host OS and localhost IP
+  # the host OS is the OS that compiles the app, not the OS that runs the app
+  const USE_QML_SERVER{.strdefine.} = "8081"
+  const isWindows = gorgeEx("wmic os get Caption").exitCode == 0
+  const isMacOS = gorge("sw_vers -productName") == "macOS"
+  const localhost = staticExec(
```

Comment:
Wouldn't this work better in a platform agnostic way?

https://nim-lang.org/docs/nativesockets.html#getHostname

Reply:
Just a suggestion though; tested the `else` variant (linux), worked fine